docs/source/contributing.rst

```diff
@@ -115,6 +115,10 @@ These are some succinct steps to set up a development environment:
 9. `Push to your fork <https://help.github.com/articles/pushing-to-a-remote/>`_.
 10. `Open a pull request! <https://help.github.com/articles/creating-a-pull-request/>`_
 
+If you experience errors complaining about `invalid command 'bdist_wheel'` during `flit install --symlink`,
```

Comment:
Should we just say `python -m pip install flit wheel` above?

Reply:
Good idea. Shorter, work better. Updated as requested.